recipes/dogpile.cache/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: dogpile.cache
+  version: 0.6.4
+
+source:
+  fn: dogpile.cache-0.6.4.tar.gz
+  url: https://files.pythonhosted.org/packages/b6/3d/35c05ca01c070bb70d9d422f2c4858ecb021b05b21af438fec5ccd7b945c/dogpile.cache-0.6.4.tar.gz 
+  md5: 66e0a6cae6c08cb1ea25f89d0eadfeb0
```

Comment:
See this example recipe here, https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L19